llvm/lib/ToolDrivers/llvm-dlltool/Options.td

```diff
@@ -12,6 +12,9 @@ def D_long : JoinedOrSeparate<["--"], "dllname">, Alias<D>;
 def d: JoinedOrSeparate<["-"], "d">, HelpText<"Input .def File">;
 def d_long : JoinedOrSeparate<["--"], "input-def">, Alias<d>;
 
+def n: JoinedOrSeparate<["-"], "n">, HelpText<"Input native .def File on ARM64EC">;
```

Comment:
Sorry to be late to the party wrt the review here, but GNU dlltool also has got a `-n` option, corresponding to `--no-delete`. So I think it would be good to make sure that the short form of the option doesn't conflict with GNU dlltool.

Reply:
Good point, I didn't notice it. I created #81847 with a fix.